src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -82,6 +83,11 @@
      */
     public static final String MSG_KEY_PARSE_ERROR = "javadoc.parse.error";
 
+    /**
+     * Message property key for the Unclosed HTML message.
+     */
+    public static final String MSG_UNCLOSED_HTML = "javadoc.unclosedHtml";
```

Comment:
please make it `MSG_UNCLOSED_HTML_TAG = "javadoc.unclosedHtmlTag"`
to make it more clear and certain.

Reply:
Done.